lib/widget/pager_indicator.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+
+class PagerIndicator extends StatefulWidget {
+  final int pagerIndicatorSize;
+  final int visibleIndex;
+
+  const PagerIndicator({
+    super.key,
+    required this.pagerIndicatorSize,
+    required this.visibleIndex,
+  });
+
+  @override
+  State<PagerIndicator> createState() => _PagerIndicatorState();
+}
+
+class _PagerIndicatorState extends State<PagerIndicator> {
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
```

Comment:
@hoangnguyen92dn Instead of using `SizedBox -> Padding`, we can use `Container` 🙏

Reply:
Oh. Nice 👍